src/main/resources/lib/github-checks/status/properties.jelly

```diff
@@ -9,17 +9,19 @@
     <f:textbox default="Jenkins"/>
   </f:entry>
 
+
   <f:entry title="${%Publish unstable builds as neutral status checks}" field="unstableBuildNeutral">
     <f:checkbox/>
   </f:entry>
 
   <f:entry title="${%Suppress log output in checks}" field="suppressLogs">
     <f:checkbox/>
   </f:entry>
-
-  <f:entry title="${%Suppress progress updates in job check}" field="skipProgressUpdates"
-           description="${%Queued, checkout and completed will still be sent}">
-    <f:checkbox/>
-  </f:entry>
-
+  <j:var set="readOnlyMode" value="${descriptor.enforceSkipProgressUpdates}" />
+  <j:if test="${!readP}">
```

Comment:
why not just do this?
```suggestion
  <j:if test="${!descriptor.enforceSkipProgressUpdates}">
```

Reply:
Same result. We built that complicated workaround because this did not work initially.